### PR TITLE
Implemented logic for DELETE method

### PR DESCRIPTION
**Pull request by suchit07-git**

### CODE REVIEW

File: pkg/database/delete.go [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package database
+
+import "log/slog"
+
+func (d Database) Delete(what string) {
+	slog.Debug("ID passed to select", what)
```
Comment (sbshah97) [inline pkg/database/delete.go:6]: nit: ID passed to delete?

File: pkg/database/delete.go [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+	data, err := d.db.Delete(what)
+	if err != nil {
+		slog.Error("Unable to delete data in table: %v", err)
```
Comment (sbshah97) [inline pkg/database/delete.go:9]: We should do a return after this line.

**Review by sbshah97 — COMMENTED** [review]
Have we tested this locally if this is working?

File: pkg/database/delete.go (hunk 1 above)
Comment (suchit07-git) [reply pkg/database/delete.go:6]: I'm sorry it's delete and not select

File: pkg/database/delete.go (hunk 2 above)
Comment (suchit07-git) [reply pkg/database/delete.go:9]: I tested it locally and it worked.

**Comment by sbshah97** [comment]
Hey can you resolve conflicts to this?

**Pull request closed without merging**